pytket/extensions/qujax/_metadata.py

```diff
@@ -1,2 +1,2 @@
-__extension_version__ = "0.2.0"
+__extension_version__ = "0.3.0"
```

Comment:
This file should no be in the git, can you remove it?

Reply:
should add to .gitignore or update my local git settings?

Reply:
It should have been in the gitignore already, if not please add it.